features/step_definitions/user_steps.rb

```diff
@@ -441,3 +441,15 @@
   @user.bio = ''
   @user.save
 end
+
+When(/^I look at user "(.*)"/) do |email|
+  @user = User.find_by_email email
+end
+
+Then(/^the tooltip (?:.*) (should not|should) be "(.*)"/) do |option, text|
```

Comment:
rather than checking the entire page, shouldn't we try to check that the tooltip occurs in the right place using `within`?